mono/utils/atomic.h

```diff
@@ -464,4 +464,12 @@ extern void InterlockedWritePointer(volatile gpointer *dst, gpointer val);
 #define InterlockedAddP(p,add) InterlockedAdd64 ((volatile gint64*)p, (gint64)add)
 #endif
 
+/* The following functions cannot be found on any platform, and thus they can be declared without further existence checks */
+
+static inline gint32
+InterlockedSubtract (volatile gint32 *dest, gint32 sub)
+{
```

Comment:
It doesn't have to match it but wouldn't it be nice as a racy operation can either be interlocked `xor` unlocked (swapping between the two could be easy)? That's at least what I had in mind when creating the function signatures. Otherwise, I would've chosen the C-style way with lowercase letters only and underscores.